README.md

```diff
@@ -26,7 +26,7 @@ To get a local copy up and running follow these simple example steps.
 # Create user
 sudo -u postgres createuser -e ced00 -s
 # Create the database
-rake db:create
+rake db:new
 # Create database Migration
 rails db:migrate
 ```
```

Comment:
**Optional**
- Please add the author section and MIT license for a professional repo.

Reply:
Hi @MahmoudBakr23 , please what do you mean by 'author section'?

Reply:
@MahmoudBakr23 It's fine. I get it now.